src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
move ColorSupplier and Random to the class field, and remove redundant empty lines from this method

Reply:
not fixed